### ISSUE TITLE
showing black image after cropping

### ISSUE THREAD

**Issue by ankush1337** [devsebastian/WonderScan#1]
when we capture image and going to cropping activity . after crop images shows black same problem as gallery when we crop any image its showing black always.

**Comment by devsebastian** [comment]
Hi @ankush1337 can you please share a video of the bug either here or at developer.devsebastian@gmail.com! also the stack trace info as well if possible!

**Comment by ankush1337** [comment]
yes sir sended.... check it

**Comment by ankush1337** [comment]
hello sir @devsebastian , please fix it sir asap

**Comment by devsebastian** [comment]
We are looking into it @ankush1337 ! As discussed if you find any solution do let us know:)

**Comment by ankush1337** [comment]
yes sir i know ...Error is on CropActivity.java where Confirm button
onclick listener . cropimagePath are null always  when we crop image thats
why its save blank image .i check it . check it sir
